src/SB/Game/zNPCFXCinematic.cpp

```diff
@@ -20,6 +25,547 @@ void zNPCFXCinematic_Init()
     }
 }
 
+void NCIN_Par_BPLANK_JET_1_Upd(const zCutsceneMgr*, NCINEntry* fxrec, S32 param)
+{
+    if (param != 0)
+    {
+        fxrec->flg_stat |= 4;
+        return;
+    }
+
+    if (fxrec->flg_stat & 2)
+    {
+        fxrec->fxdata.pardata.emitter = zParEmitterFind("PAREMIT_BPLANK_JET_1");
+
+        if (fxrec->fxdata.pardata.emitter->tasset)
+        {
+            if (((U8*)fxrec->fxdata.pardata.emitter->tasset)[9] != 15)
+            {
+                fxrec->fxdata.pardata.emitter = NULL;
+            }
+        }
+    }
+
+    if (fxrec->fxdata.pardata.emitter == NULL)
+    {
+        fxrec->flg_stat |= 4;
+        return;
+    }
+
+    F32 & float_accum = *(F32*)((U8*)fxrec + 40);
```

Comment:
Try to write this in terms of a struct member and not an indefinite char offset from the fxrec pointer. It should look like your other code (e.g. fxrec->flg_stat, fxrec->fxdata).